README.md

```diff
@@ -54,6 +54,7 @@ CREATE OR REPLACE TABLE <my-project>.<my-dataset>.antipattern_output_table (
   query STRING,
   recommendation ARRAY<STRUCT<name STRING, description STRING>>,
   slot_hours FLOAT64,
+  optimized_sql STRING,
```

Comment:
need to add the optimized_sql field in the bq example doc as well ./EXAMPLES.md#bq-table---bq-table